Purchases/Purchasing/StoreKitAbstractions/SK2StoreProductDiscount.swift

```diff
@@ -27,6 +27,15 @@ internal struct SK2StoreProductDiscount: StoreProductDiscountType {
         self.price = sk2Discount.price
         self.paymentMode = paymentMode
         self.subscriptionPeriod = subscriptionPeriod
+
+        switch sk2Discount.type {
+        case SK2ProductDiscount.OfferType.introductory:
+            self.type = .introductory
+        case SK2ProductDiscount.OfferType.promotional:
+            self.type = .promotional
+        default:
```

Comment:
Can you do `@unknown`?

Reply:
`@unknown` doesn't work here because these aren't enums but are structs 😬